tst/regression/utils/test_case.py

```diff
@@ -89,9 +89,7 @@ def __init__(self, run_test_path, **kwargs):
         try:
             parthenon_path = os.path.realpath(__file__)
             idx = parthenon_path.rindex("/parthenon/")
-            self.parameters.parthenon_path = os.path.join(
-                parthenon_path[:idx], "parthenon"
-            )
+            self.parameters.parthenon_path = parthenon_path[: idx + 10]
```

Comment:
FYI this fixes an edge case where the path would be wrong if `parthenon` is checked out to `/parthenon`